projects/core/src/lib/users/quota/quota-info/quota-info.component.ts

```diff
@@ -0,0 +1,79 @@
+import {ChangeDetectorRef, Component, OnDestroy} from '@angular/core';
+import {ProjectService} from '../../../project/project.service';
+import {BehaviorSubject, combineLatest, mergeMap, of, Subscription} from 'rxjs';
+import {UserService} from '../../user.service';
+import {Quota} from '../quota.model';
+import {BackendService} from '../../../backend/backend.service';
+
+@Component({
+    selector: 'geoengine-quota-info',
+    templateUrl: './quota-info.component.html',
+    styleUrls: ['./quota-info.component.scss'],
+})
+export class QuotaInfoComponent implements OnDestroy {
+    sessionQuota: Quota | undefined;
+    updateQuota$: BehaviorSubject<void> = new BehaviorSubject<void>(undefined);
+    sessionSubscription: Subscription | undefined;
+    layerSubscription: Subscription | undefined;
+    plotSubscription: Subscription | undefined;
+
+    constructor(
+        protected readonly userService: UserService,
+        protected readonly projectService: ProjectService,
+        protected readonly backendService: BackendService,
+        protected readonly changeDetectorRef: ChangeDetectorRef,
+    ) {
+        this.sessionSubscription = combineLatest([
+            this.userService.isBackendFeatureEnabled('pro'),
+            this.userService.getSessionStream(),
+            this.updateQuota$,
+        ])
+            .pipe(
+                mergeMap(([isPro, _session, _update]) => {
+                    if (!isPro) {
+                        return of(undefined);
+                    }
+                    return this.userService.getSessionQuota();
+                }),
+            )
+            .subscribe((quota) => {
+                if (quota) {
+                    this.sessionQuota = Quota.fromDict(quota);
+                } else {
+                    this.sessionQuota = undefined;
+                }
+                this.changeDetectorRef.markForCheck();
+            });
+
+        this.layerSubscription = this.projectService.getLayerStream().subscribe((_layer) => {
+            this.refreshQuota();
+        });
+        this.plotSubscription = this.projectService.getPlotStream().subscribe((_plot) => {
+            this.refreshQuota();
+        });
+    }
+
+    ngOnDestroy(): void {
+        this.sessionSubscription?.unsubscribe();
+        this.layerSubscription?.unsubscribe();
+        this.plotSubscription?.unsubscribe();
+    }
+
+    get progressColor(): string {
+        if (!this.sessionQuota) {
+            return 'gray';
```

Comment:
I don't think this is valid.

I guess progressColor() should return `ThemePalette` instead.

`import {ThemePalette} from '@angular/material/core';`

Reply:
[58b8758](https://github.com/geo-engine/geoengine-ui/pull/586/commits/58b875872fb50dde85173efc2b933ff9ade19431)

Reply:
the progessbar was removed [e46564b](https://github.com/geo-engine/geoengine-ui/pull/586/commits/e46564b203836f6ce2baf1a8d4eec605fc520333)